bucket/helm.json

```diff
@@ -1,24 +1,32 @@
 {
-    "version": "3.19.2",
-    "description": "The package manager for Kubernetes",
+    "version": "4.0.0",
+    "description": "The package manager for Kubernetes.",
     "homepage": "https://helm.sh",
     "license": "Apache-2.0",
     "architecture": {
         "64bit": {
-            "url": "https://get.helm.sh/helm-v3.19.2-windows-amd64.zip",
-            "hash": "8b3ea37bcd7c5eda68eadd278690498fb886a17ec541b449216ab3b45748600e",
+            "url": "https://get.helm.sh/helm-v4.0.0-windows-amd64.zip",
+            "hash": "0f9a8c891b8d908a37fbb68f12dea92b633eb29e49070bd650f5760a1a99aa8d",
             "extract_dir": "windows-amd64"
+        },
+        "arm64": {
+            "url": "https://get.helm.sh/helm-v4.0.0-windows-arm64.zip",
+            "hash": "f3ff262427547cc1b1dc3356d587ed8ffaa23f2abf24bc06660a350b9b7925f9",
+            "extract_dir": "windows-arm64"
         }
     },
     "bin": "helm.exe",
     "checkver": {
-        "url": "https://github.com/kubernetes/helm/releases",
-        "regex": "/tag/v(3\\.[\\d.]+)\""
+        "url": "https://github.com/helm/helm/releases",
+        "regex": "\"/helm/helm/releases/tag/v([\\d]+\\.[\\d]+\\.[\\d]+)\""
```

Comment:
generally speaking, I believe "/helm/helm/releases" is not required to match the release here, match the tag should be enough to get the releases. Simply matching "/tag/v([\\d]+\\.[\\d]+\\.[\\d]+)" should be enough.

Reply:
This regex matches the GitHub generated content for the release tag link and makes the likelihood of finding a match in the release description significantly lower. The regex you've provided wouldn't work as it would pick up pre-release versions due to the removal of the `\"` at the end.

Reply:
I didn't mean to remove the trailing slash, that's not align with what I said and it's an unintentional error.

Reply:
That's not true. GitHub matcher in this case is only used to get the latest release's tag that is not marked as pre release. It will then pass the version variable to the autoupdate part so autoupdate can construct the url correctly.